ci-droid-tasks-consumer-infrastructure/src/test/resources/application-test.yml

```diff
@@ -23,5 +23,8 @@ cidroid-behavior:
     limitInDays: 180
 # whenPullRequestEvent:
   bestPracticeNotifier.enabled: true
+  maxFilesInPRNotifier.enabled: true
   patternToResourceMapping:
     "**/*Dto.java": "https://someLinkToSomeResource/ci-droid-best-practices/noDtoClass.md"
+    maxFilesInPr: 10
```

Comment:
same as for application.yml

can we also have a test that actually reads these properties, and therefore makes sure they are at the expected place in the config file ? Because I removed these locally, and build is still successful.

Reply:
Thanks for your comments. Corrected the configurations.To test the configurations,  I can not think of anything easier than adding IT. Do you have any suggestion?

Reply:
I am afraid you're right. We can keep it simple though - if you're able to launch CiDroidTasksConsumerApplication with CiDroidTasksConsumerAutoConfiguration with the "enabled" config , checking that the bean is indeed loaded could be enough - an error in config would either prevent the application from starting, or the bean wouldn't load

Reply:
A test case for configuration is added. Can you please check ?